src/analog/m2kanalogin_impl.hpp

```diff
@@ -180,6 +181,9 @@ class M2kAnalogInImpl : public M2kAnalogIn
 
 	int convertVoltsToRawVerticalOffset(ANALOG_IN_CHANNEL channel, double vertOffset);
 	double convertRawToVoltsVerticalOffset(ANALOG_IN_CHANNEL channel, int rawVertOffset);
+
+	void setCalibrateHDL(std::string calibrate_val);
```

Comment:
could you please provide a short description of this new function, in which fw version was this added and why?